tests/test_lua_property.cpp

```diff
@@ -0,0 +1,260 @@
+        defer { LogMgr.resetMessageCounters(); };
+
+        global::scriptEngine->queueScript({
+            .code = "return openspace.property('other-name')",
+            .callback = [](ghoul::Dictionary d) {
+                REQUIRE(d.size() == 1);
+                REQUIRE(d.hasKey("1"));
+                REQUIRE(d.hasValue<ghoul::Dictionary>("1"));
+                ghoul::Dictionary e = d.value<ghoul::Dictionary>("1");
+                CHECK(e.size() == 0);
+            }
+        });
+
+        triggerScriptRun();
+    }
+    
+    {
+        LogMgr.resetMessageCounters();
+        defer { LogMgr.resetMessageCounters(); };
+
+        global::scriptEngine->queueScript({
+            .code = "return openspace.property('base.other-name')",
+            .callback = [](ghoul::Dictionary d) {
+                REQUIRE(d.size() == 1);
+                REQUIRE(d.hasKey("1"));
+                REQUIRE(d.hasValue<ghoul::Dictionary>("1"));
+                ghoul::Dictionary e = d.value<ghoul::Dictionary>("1");
+                CHECK(e.size() == 0);
+            }
+        });
+
+        triggerScriptRun();
+    }
+}
+
+TEST_CASE("Property: Multiple", "[property]") {
+    PropertyOwner owner = PropertyOwner({ "base" });
+    global::rootPropertyOwner->addPropertySubOwner(owner);
+    defer { global::rootPropertyOwner->removePropertySubOwner(owner); };
+    FloatProperty p1 = FloatProperty(Property::PropertyInfo("p1", "a", "b"), 1.f);
+    owner.addProperty(p1);
+    FloatProperty p2 = FloatProperty(Property::PropertyInfo("p2", "a", "b"), 1.f);
+    owner.addProperty(p2);
+
+    {
+        LogMgr.resetMessageCounters();
+        defer { LogMgr.resetMessageCounters(); };
+
+        global::scriptEngine->queueScript({
+            .code = "return openspace.property('base.*')",
+            .callback = [&p1, &p2](ghoul::Dictionary d) {
+                REQUIRE(d.size() == 1);
+                REQUIRE(d.hasKey("1"));
+                REQUIRE(d.hasValue<ghoul::Dictionary>("1"));
+                ghoul::Dictionary e = d.value<ghoul::Dictionary>("1");
+                REQUIRE(e.size() == 2);
+
+                // Sorting the results as the property return order is undefined
+                std::array<std::string, 2> r;
+                {
+                    REQUIRE(e.hasKey("1"));
+                    REQUIRE(e.hasValue<std::string>("1"));
+                    const std::string v = e.value<std::string>("1");
+                    r[0] = v;
+                }
+                {
+                    REQUIRE(e.hasKey("2"));
+                    REQUIRE(e.hasValue<std::string>("2"));
+                    const std::string v = e.value<std::string>("2");
+                    r[1] = v;
+                }
+
+                std::sort(r.begin(), r.end());
+                CHECK(r[0] == p1.uri());
+                CHECK(r[1] == p2.uri());
+            }
+        });
+
+        triggerScriptRun();
+    }
+}
+
+TEST_CASE("Property: Multiple / 2", "[property]") {
```

Comment:
What do you mean by `/ 2` here?  Please change to a more understandable name :) 

Also, Missing examples with tag, and wildcard in another place than at the end of the uri

Reply:
It's the second type of test for multiple, but no other useful difference that would be expressable with a simple name.

Bringing it into the same naming scheme as used in other places (https://github.com/OpenSpace/codegen/blob/master/tests/parsing_luawrapper/parsing_luawrapper_fail.cpp#L210).